paasta_tools/cli/cmds/status.py

```diff
@@ -83,6 +84,8 @@
 from paasta_tools.utils import remove_ansi_escape_sequences
 from paasta_tools.utils import SystemPaastaConfig
 
+KUBERNETES_INSTANCE_TYPE = "kubernetes"
```

Comment:
should this be defined in `paasta_tools.instance.kubernetes`? this file doesn't really seem like the best place for it